paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlSyncDatabaseAction.java

```diff
@@ -106,16 +116,49 @@ public class MySqlSyncDatabaseAction implements Action {
             boolean ignoreIncompatible,
             Map<String, String> catalogConfig,
             Map<String, String> tableConfig) {
+        this(
+                mySqlConfig,
+                warehouse,
+                database,
+                ignoreIncompatible,
+                null,
+                null,
+                null,
+                null,
+                catalogConfig,
+                tableConfig);
+    }
+
+    MySqlSyncDatabaseAction(
+            Map<String, String> mySqlConfig,
+            String warehouse,
+            String database,
+            boolean ignoreIncompatible,
+            @Nullable String tablePrefix,
+            @Nullable String tableSuffix,
+            @Nullable String includingTable,
+            @Nullable String excludingTable,
+            Map<String, String> catalogConfig,
+            Map<String, String> tableConfig) {
         this.mySqlConfig = Configuration.fromMap(mySqlConfig);
         this.warehouse = warehouse;
         this.database = database;
         this.ignoreIncompatible = ignoreIncompatible;
+        this.tablePrefix = tablePrefix == null ? "" : tablePrefix;
+        this.tableSuffix = tableSuffix == null ? "" : tableSuffix;
+
+        checkArgument(
+                includingTable == null || excludingTable == null,
+                "Cannot specify including tables and excluding tables at the same time.");
```

Comment:
Maybe we can support both includingTable and excludingTable, it looks not complicated.